live-build/misc/ansible-roles/appliance-build.virtualization-development/tasks/main.yml

```diff
@@ -87,3 +87,11 @@
     password:
       "{{ lookup('env', 'APPLIANCE_PASSWORD') | password_hash('sha512') }}"
   no_log: true
+
+- apt:
+    name: "libnss3*"
```

Comment:
What are the tradeoffs of installing the `libnss3*` packages here rather than listing these packages as dependencies of the Virtualization Engine package?

Reply:
+1 .. I think this is the wrong place to do this, but perhaps I'm overlooking something.

Reply:
Now I see this is being done in the development role.. if we don't want these packages on the qa and/or customer image, then I take back my prior statement; this is probably the right place to install the packages.

If that's the case, why install with a glob of packages, rather than being more explicit about the packages we require?

Reply:
I see that using the wildcard is confusing. This is meant to install the -dev and -dbg packages, which are only needed for development. I'll make this explicit.